### PR TITLE
LV522: Chances Changes

### PR DESCRIPTION
**Pull request by spartanbobby**
# About the pull request

This PR makes numerous Changes to LV522 as well as adds more props to ease of mapper use

# Explain why it's good for the game

The areas changes and reasons why as are follows
**SW Reactor:** Entirely removed and replaced with a much more open containers area for the xenos to build up and defend, this area is now the linchpin of the map as marines have to push though it to get to other flanks inside the reactor meaning the xeno players should now have a much better understanding of where they need to defend instead of prior problems with the map of them being hard flanked and losing at 12:26

**LZ1:** LZ1 was moved slightly more north in an attempt to remove some deadspace and make the path from the front to the FOB slightly less long, more LZ1 tunnels and ways inside the FOB were added for xeno flankers aswell as LZ1 being locked down until the marines push a button to open it up

**Colony admin** A sensor tower was added to colony admin to encourage marines to go over there and investigate, along with a lockdown to the area being added 

**LZ3** a FORECON prop LZ housing the Tornado was re-added after being removed in an attempt to downsize the map, basically I figured out where I could put it

**props:** Alot of instanced props for the map were made into actual items such as, bedrolls and folded bedrolls, FORECON dogtags, used flares, jerry cans, used bandages and some weird gameboy thing

Sprites: https://i.imgur.com/avi2fUo.png
FORECON was always made to have a patch it was one of those things I wanted but never got, luckily while making this PR I was able to look into it and get the old sprite from tri to finish up myself with onmobs

FORECON Balance changes: The M39 being in the primary weapon slot is more of a "fuck you" to people playing the roll than just unlucky RNG that is workaround able moving it to the secondary pool lets the FORECON play around with better weapons to survive with and the M39 extended magazines means it's one of the only weapons FORECON spawn with that has a decent amount of ammo

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:SpartanBobby
add: Made some instanced props on LV522 actual items for mapper ease of use
maptweak: tweaked LV522, removing the SW reactor and replacing it with a much more open container area once used as the FORECONS FOB
maptweak: tweaked LZ1 on LV522 making it start locked down and be slightly more north along with some extra tunnels and flanking routes for the xenos
maptweak: LV522, added a lockdown and moved the sensor tower to colony admin
maptweak: re-added the prop LZ in the NE of the colony
maptweak: redistributed LV522 mining metal spawns to be more spread out
maptweak: removed building color outlines from LV522 that were there to help with navigation during Test merges since the map has been out for long enough for the majority to properly navigate it
tweak: M39 removed from FORECON primary weapon pool and added to secondary weapon pool along with extended mags instead of normal
add: Added FORECON Patches for the survivors on LV522 sprites made by Triiodine while onmobs were made by myself with help from Kugamo
fix: examing a uniform no longer tells you that it has "an USCM/FORECON/Falling falcons patch" instead it says "a patch"
add: updated the USCM FORECON uniform name and description 
/:cl:

### CODE REVIEW

**Review by harryob — APPROVED** [review]
code looks good

**Comment by spartanbobby** [comment]
Going to need another code review and sprite review because I managed to get the stuff for the FORECON patches

File: code/modules/gear_presets/forcon_survivors.dm [hunk 1]
```diff
@@ -41,31 +43,32 @@
...
-			H.equip_to_slot_or_del(new /obj/item/weapon/gun/smg/m39(H), WEAR_L_HAND)
-			H.equip_to_slot_or_del(new /obj/item/ammo_magazine/smg/m39(H), WEAR_IN_BACK)
-			H.equip_to_slot_or_del(new /obj/item/ammo_magazine/smg/m39(H), WEAR_IN_BACK)
 
 /datum/equipment_preset/survivor/forecon/add_survivor_weapon_pistol(mob/living/carbon/human/H)
 	return
 
 /datum/equipment_preset/survivor/forecon/proc/add_forecon_weapon_pistol(mob/living/carbon/human/H)
-	var/random_pistol = rand(1,3)
+	var/random_pistol = rand(1,6)
```
Comment (harryob) [inline code/modules/gear_presets/forcon_survivors.dm:59]: does 5 get no pistol

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by Agameofscones — APPROVED** [review]
nice patch 😎

**Pull request merged**